content/en/blog/argo-insurance-workflow/index.md

```diff
@@ -0,0 +1,126 @@
++++
+title = "Fission Function Orchestration with Argo Workflows"
+date = "2022-09-21T10:15:00+05:30"
+author = "Neha Gupta"
+description = "Multiple Fission function orchestration using Argo Workflows"
+categories = ["Tutorials"]
+type = "blog"
+images = ["images/featured/fission-zapier-featured.png"]
++++
+
+Fission Functions allow user to perform one logical task. To group multiple task together such as one function is dependent on other we can use Argo Workflows.
+Argo Workflows is an open source container-native workflow engine with a feature to create DAGs i.e. running task sequentially, in parallel and with dependencies. We will try to develop a simple Insurance Eligibility program which will take different input and calculate insurance installment on basis of the inputs.
+
+### Understanding the Insurance Calculator
+
+The small application/Workflow will take basic information from the user and check the eligibility for the Insurance. Below are the input and different tasks which are performed to calculating the above:
+
+* Take first name of the user
+* Take last name of the user
+* Create full name using the above two.
+* Input age and monthly salary.
+* On basis of predefined criteria i.e. ,
+  * if age is below a certain age and yearly salary is above some numbers, user lies in a low risk criteria
+  * otherwise he/she falls in high risk criteria. And the installments varies accordingly.
+* We print the above calculated output to the user.
```

Comment:
The current steps are confusing from an end user point of view. Let us refine this section into two parts.

In the first part, list the steps that the end user will perform, something like below:
- Enter name
- Enter age and monthly Salary
- Click Ok

In the second, mention how it works in the background, what services are called, workflow triggered etc (_basically explain this flow diagram this is shown_) and if possible add the screenshot/UI of the application

Reply:
done